scripts/qa_crowbarsetup.sh

```diff
@@ -615,9 +615,13 @@ function onadmin_prepare_sles12sp1_other_repos
 
 function onadmin_prepare_sles_other_repos
 {
-    for repo in SLES$slesversion-{Pool,Updates,LTSS-Updates}; do
+    for repo in SLES$slesversion-{Pool,Updates}; do
         add_mount "repos/$arch/$repo" "$tftpboot_repos_dir/$repo"
     done
+    # FIXME LTSS repo is still empty for SOC8, do not add it as zypper does not like empty repos
```

Comment:
Is this supposed to be SOC9 (like the commit message says)?

Reply:
@cmurphy fixed the commit message instead - thanks!